internal/cmd/get/get_test.go

```diff
@@ -71,3 +74,56 @@ func Test_initialiseProjectInCurrentDir(t *testing.T) {
 		})
 	}
 }
+
+func Test_run(t *testing.T) {
```

Comment:
the test's name doesn't say much about what it tests. How about `Test_Detecting_Already_Registered_Command`?